### ISSUE TITLE
Special Projects

### ISSUE THREAD

**Issue by bethanyriggle** [afsc-gap-products/GOA-Survey-Planning#80]

**Comment by MargaretSiple-NOAA** [comment]
For next year/this year:  

I think this as well as #78, #79, and #80 can be consolidated for task list 2025 and beyond. Many of them are processed in the same `navmaps` script so it makes sense to assign everything to the same team. That team would need access to 32-bit R (on their own machine or on the special VM that Sean set up for this) and will need to have the `navmaps`  R package installed. There are notes about data sources in the [`navmaps` README](https://github.com/afsc-gap-products/navmaps).

We can co-opt the task checklist from the EBS for that task. From EBS, this task list (in order of operation) is:

### Issue Description
Use afsc-gap-products/navmaps to create electronic survey charts. Request access to VM AKC0SS-VU-241 to generate Globe layers using 32-bit R.

### Tasklist

- [ ] Get shapefiles/csvs for special collections and add to EBS script
- [ ] Update protected species (SSL, NPRW, sea otter, spectacled eider) shapefiles in the package if needed
- [ ] Create EBS files for TimeZero using R script in navmaps
- [ ] Create EBS files for Globe using R script in navmaps
- [ ] Create EBS survey shapefiles for ArcMap using R script in navmaps
- [ ] Test files in TimeZero and Globe
- [ ] Add TimeZero and Globe files to the Survey App

**Comment by coleary-noaa** [comment]
agreed. I think we can combine those tasks into a "navmaps" task and then just have an alert arcmap charts person that the package was run and their layers are there if they need.

**Comment by MarkZimmermann-NOAA** [comment]
Thanks Alex,
For this situation, the geographic instructions (Leg 1 or Leg 3) for those
particular special projects should probably just go on the deck placards.

Generally the Lat/Long boxes are best presented on the station charts
rather than deck placards, but the info of when the boat crosses inside or
outside of the boxes needs to be verbally relayed by the FPC to deck folks.

Yay!
Can we cross this one off of the to-do list?

Thanks again,
Mark

**Comment by AlexandraDowlin-NOAA** [comment]
@MarkZimmermann-NOAA Yes! Let's close this issue!
Thanks everyone.
